src/components/pages/Dashboard/RenderDashboardPage.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { connect } from 'react-redux';
 import MyProfileComponent from '../../common/MyProfileComponent';
 import MetricsFilterBar from '../../common/MetricsFilterBar/MetricsFilterBar';
-import GraphsPlaceholder from '../../common/GraphsPlaceholder/GraphsPlaceholder';
+// import GraphsPlaceholder from '../../common/GraphsPlaceholder/GraphsPlaceholder';
```

Comment:
Same thing here, we'll probably use it later so the comment is ok with me but some other people might not like it.